### PR TITLE
[O11y][MySQL] Update system test cases

### LINKED ISSUE #4372: Write system test cases for performance data stream of Mysql
Add system test cases for performance data stream of Mysql as a standard practice.

### PR DESCRIPTION
**Pull request by aliabbas-elastic**
- Enhancement

## What does this PR do?

- This PR updates system test cases for the MySQL package.
- System tests are done in the supported versions mentioned in the `variants.yml` due to this [issue](https://github.com/elastic/beats/issues/25069).
Conversation for the same:- [Link](https://github.com/elastic/integrations/issues/4372#issuecomment-1567776226)
- Updated `sample_event.json` for the performance data stream.

## Checklist

- [x] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.

## How to test this PR locally

- Clone integrations repo.
- Install elastic-package locally.
- Start elastic stack using elastic-package.
- Move to integrations/packages/mysql directory.
- Run the following command to run the system tests. elastic-package test system

## Related issues

- Closes #4372

image
image
image

### CODE REVIEW

**Review by harnish-elastic — APPROVED** [review]
LGTM!

File: packages/mysql/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.12.1"
```
[the remark this answers is not in the document]
Comment (aliabbas-elastic) [reply packages/mysql/changelog.yml:2]: Isn't this a change we are making which has no end-user impact? It's a minor change, right?
Still, if you think we need to bump up the version `1.13.0` let me know your thoughts.

**Review by ritalwar — APPROVED** [review]
LGTM!

**Pull request merged**